testsuite/policy/authorization/auth_config.py

```diff
@@ -76,13 +74,12 @@ def remove_all_hosts(self):
 
     def wait_for_ready(self):
         """Waits until authorization object reports ready status"""
-        with oc.timeout(90):
-            success, _, _ = self.self_selector().until_all(
-                success_func=lambda obj: len(obj.model.status.conditions) > 0
-                and all(x.status == "True" for x in obj.model.status.conditions)
-            )
-            assert success, f"{self.kind()} did not get ready in time"
-            self.refresh()
+        success = self.wait_until(
+            lambda obj: len(obj.model.status.conditions) > 0
+            and all(x.status == "True" for x in obj.model.status.conditions)
```

Comment:
This apparently works well but would not it be better to specifically check the `Enforced=True` ? I think that in context of Policies the "ready" actually means enforced, doesn't it?

```
lambda obj: all(x.status == "True" for x in obj.model.status.conditions)
            and any(x.type == "Enforced" for x in obj.model.status.conditions)
```

Reply:
Authpolicy does have Enforced but AuthConfig is not a policy so I am not sure about that

Reply:
Crap, sorry, I completely missed the fact that this is about AuthConfig, not AuthPolicy. You are right, only policies are enforced.